.goreleaser.yaml

```diff
@@ -0,0 +1,28 @@
+project_name: ingress-nginx
+release:
+  github:
+    owner: kubernetes
+    name: ingress-nginx
+builds:
+  - id: ingress-nginx
+    goos:
+    - darwin
+    - linux
+    - windows
+    goarch:
+    - arm64
+    - amd64
+    env:
+      - CGO_ENABLED=0
```

Comment:
Is there a style guide for this indentation? (Or is there a formatter that this project would normally use for yaml?)

`goos:`/`darwin` is at least inconsistent with `env:`/`CGO_ENABLED=0`, and probably with `builds:`/`id:`.

Reply:
Fair point, should be more consistent now.